zstd_vendor/CMakeLists.txt

```
@@ -4,12 +4,14 @@ project(zstd_vendor)
 
 find_package(ament_cmake REQUIRED)
 
+set(CMAKE_MODULE_PATH "${CMAKE_MODFULE_PATH}" "${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules")
```

Our usual pattern here is to use the `list` command to insert the module path:

```suggestion
list(INSERT CMAKE_MODULE_PATH 0 "${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules")
```